src/touch.js

```diff
@@ -8,7 +8,10 @@ var
 
 var
 	Dom = require('./dom'),
-	Job = require('./job');
+	Job = require('./job'),
+	Control = require('./Control');
+
+var rAF = global.requestAnimationFrame;
```

Comment:
We have a wrapper around rAF to prevent issues where it doesn't exist on the platform:  animation.requestAnimationFrame.